src/main/java/liquibase/parser/ext/LintAwareChangeLogParser.java

```diff
@@ -115,16 +127,49 @@ protected void loadConfig(ResourceAccessor resourceAccessor) {
         }
     }
 
-    void checkDuplicateIncludes(String physicalChangeLogLocation, Config config) throws ChangeLogParseException {
+    private void checkForFilesNotIncluded(Config config, ResourceAccessor resourceAccessor) throws ChangeLogParseException {
+        RuleConfig ruleConfig = config.getRules().get("file-not-included").stream()
```

Comment:
Feels like this method could do with pushing into `Config`?

Reply:
Done